elephant/test/test_surrogates.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+"""
+docstring goes here.
+
+:copyright: Copyright 2014 by the Elephant team, see AUTHORS.txt.
+:license: Modified BSD, see LICENSE.txt for details.
+"""
+
+import unittest
+import elephant.surrogates as surr
+import numpy as np
+import quantities as pq
+import neo
+
+
+class SurrogatesTestCase(unittest.TestCase):
+
+    def test_spike_dithering_output_format(self):
+
+        st = neo.SpikeTrain([90, 150, 180, 350] * pq.ms, t_stop=500 * pq.ms)
+
+        nr_surr = 2
+        dither = 10 * pq.ms
+        surrs = surr.spike_dithering(st, dither=dither, n=nr_surr)
+
+        self.assertEqual(type(surrs), list)
+        self.assertEqual(len(surrs), nr_surr)
+
+        for surrog in surrs:
+            self.assertEqual(type(surrs[0]), neo.SpikeTrain)
```

Comment:
can use `self.assertIsInstance(surrs[0], neo.SpikeTrain)` to perform the same test but with a more informative error message if it fails.